internal/tsoptions/tsconfigparsing.go

```diff
@@ -119,8 +121,8 @@ func parseOwnConfigOfJsonSourceFile(
 	basePath string,
 	configFileName string,
 ) (*parsedTsconfig, []*ast.Diagnostic) {
-	options := getDefaultCompilerOptions(configFileName)
-	// var typeAcquisition *compiler.TypeAcquisition
+	compilerOptions := getDefaultCompilerOptions(configFileName)
+	var typeAcquisition *core.TypeAcquisition
```

Comment:
This should probably be `getDefaultTypeAcquistion` https://github.com/microsoft/TypeScript/blob/main/src/compiler/commandLineParser.ts#L3545   .